common/Kconfig

```diff
@@ -964,6 +964,14 @@ config SPL_FIOVB
 	 This option enables compilation of Foundries.IO verified boot
 	 operations
 
+config SCP03
+	bool "Build SCP03 - Secure Channel Protocol O3 - controls"
+	depends on OPTEE
+	default y
```

Comment:
Should we really default to y? I would think this should be an opt-in feature instead.

Reply:
yes. sorry I took this shortcut but forgot to remove it (I thought this was an urgent PR so I kind of rushed it)...will remove